lib/matplotlib/animation.py

```diff
@@ -1222,6 +1222,14 @@ def _setup_blit(self):
         # axes
         self._blit_cache = dict()
         self._drawn_artists = []
+        def remove_from_cache(ax):
+            try:
+                del self._blit_cache[ax]
```

Comment:
Could replace the whole block with:
```python
self._blit_cache.pop(ax, None)
```

Reply:
@dopplershift,
I really like your proposal. The second argument of dict.pop did not come to my mind. We could make my proposal much more concise:
```
        self._drawn_artists = []
        for ax in self._fig.axes:
            ax.callbacks.connect('xlim_changed',
                                 lambda ax: self._blit_cache.pop(ax, None))
            ax.callbacks.connect('ylim_changed',
                                 lambda ax: self._blit_cache.pop(ax, None))
        self._resize_id = self._fig.canvas.mpl_connect('resize_event',
                                                       self._handle_resize)
```
This was my first contribution to matplotlib. Shall I file another pull request for this modification?

Reply:
@onatt, no just modify this PR, add a new commit on your machine, and then push...  https://matplotlib.org/devel/index.html